src/server/api/organization.js

```diff
@@ -32,9 +32,21 @@ export const resolvers = {
       return r.table('opt_out')
         .getAll(organization.id, { index: 'organization_id' })
     },
-    people: async (organization, { role, campaignId }, { user }) => {
+    people: async (organization, { role, campaignId, offset }, { user }) => {
       await accessRequired(user, organization.id, 'SUPERVOLUNTEER')
-      return buildUserOrganizationQuery(r.knex.select('user.*'), organization.id, role, campaignId)
+      const query = buildUserOrganizationQuery(
```

Comment:
did you and @erikkirkstein huddle about changing this to most recently created?

Reply:
Yeah. He preferred reverse by created-at, since most issues are related to new users.  It does make hunting for a specific user annoying but
1. that's only an issue for teams with >200 anyway, which is already a pretty small set of orgs
2. looking forward to Justin's PR that allows you to do a search